### ISSUE TITLE
Add ability to control systemd user instance services

### ISSUE THREAD

**Issue by firecat53** [ansible/ansible-modules-core#1479]
### Issue Type:

Feature Idea
### Ansible Version:

1.9.1
### Ansible Configuration:

N/A
### Environment:

Linux - (systemd)
### Summary:

It would be useful to be able to control systemd --user services from the
service module. I'm still researching how/if this can be done from the command
line...those aren't the easiest search terms to narrow down.
### Steps To Reproduce:

Possible usage:

```
service:
  name: syncthing
  user: firecat53
  state: restarted
```
### Expected Results:

N/A
### Actual Results:

N/A

**Comment by bcoca** [comment]
Fixed by the new systemd module.

If you continue seeing any problems related to this issue, or if you have any further questions, please let us know by stopping by one of the two mailing lists, as appropriate:

https://groups.google.com/forum/#!forum/ansible-project - for user questions, tips, and tricks
https://groups.google.com/forum/#!forum/ansible-devel - for strategy, future planning, and questions about writing code
Because this project is very active, we're unlikely to see comments made on closed tickets, but the mailing list is a great way to ask questions, or post if you don't think this particular issue is resolved.

Thank you!

**Comment by runcom** [comment]
@bocca any idea when will this be available? https://docs.ansible.com/ansible/systemd_module.html reports that it's available in version `2.2` - what does this version stands for? I have 2.1.0.0 and I cannot find a 2.2 release in the ansible Releases github page

**Comment by denisdefreyne** [comment]
@runcom Ansible 2.2 is not out (it’s a release candidate atm). See https://releases.ansible.com/ansible/.
